be/src/formats/parquet/encoding.cpp

```diff
@@ -44,6 +45,18 @@ struct TypeEncodingTraits<type, tparquet::Encoding::RLE_DICTIONARY> {
     }
 };
 
+template <tparquet::Type::type type>
+struct TypeEncodingTraits<type, tparquet::Encoding::RLE> {
+    static Status create_decoder(std::unique_ptr<Decoder>* decoder) {
+        decoder->reset(new RleDecoder<typename PhysicalTypeTraits<type>::CppType>());
```

Comment:
`RleDecoder` is not deriving from `Decoder`, so we should create another class deriving from `Decoder` and encapsulate `RleDecoder`